Library/Formula/sshguard.rb

```diff
@@ -12,6 +12,20 @@ class Sshguard < Formula
     url "http://sourceforge.net/p/sshguard/bugs/_discuss/thread/3d94b7ef/c062/attachment/sshguard.c.diff"
     sha1 "68cd0910d310e4d23e7752dee1b077ccfe715c0b"
   end
+  
+  # Fix parsing problem (chokes on "via")
+  # See:
+  # http://sourceforge.net/p/sshguard/mailman/message/33330543/
+  patch do
+    url "https://bitbucket.org/sshguard/sshguard/commits/fc01ad308c10ceb4164e23967b4713b9e7e533d7/raw/"
+    sha256 "b582206fd286a89b15f41c44460de3c33386563a5c01876f85b88018feea786c"
+  end
+
+  # Alternative source: Discussion thread on this error
```

Comment:
Done. I don't know how chatty you guys like your formulae :)

On Mon, Apr 6, 2015 at 10:33 AM, Dominyk Tiller <notifications@github.com>
wrote:

> In Library/Formula/sshguard.rb
> <https://github.com/Homebrew/homebrew/pull/38080#discussion_r27800129>:
>
> > @@ -12,6 +12,20 @@ class Sshguard < Formula
> >      url "http://sourceforge.net/p/sshguard/bugs/_discuss/thread/3d94b7ef/c062/attachment/sshguard.c.diff"
> >      sha1 "68cd0910d310e4d23e7752dee1b077ccfe715c0b"
> >    end
> > +
> > +  # Fix parsing problem (chokes on "via")
> > +  # See:
> > +  # http://sourceforge.net/p/sshguard/mailman/message/33330543/
> > +  patch do
> > +    url "https://bitbucket.org/sshguard/sshguard/commits/fc01ad308c10ceb4164e23967b4713b9e7e533d7/raw/"
> > +    sha256 "b582206fd286a89b15f41c44460de3c33386563a5c01876f85b88018feea786c"
> > +  end
> > +
> > +  # Alternative source: Discussion thread on this error
>
> You can probably kill this commented out block entirely. You've added the
> mailing list discussion as a link above the commit, so we'll know where to
> find it if necessary :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/38080/files#r27800129>.
>